doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -358,60 +463,81 @@ PyROS requires the user to supply one local and one global NLP solver to be used
   >>> global_solver = pyo.SolverFactory('baron')
 
 .. note::
-    Additional solvers to be used as backup can be designated during the ``solve`` statement via the config options ``backup_local_solvers`` and ``backup_global_solvers`` presented above.
-
-The final step in solving a model with PyROS is to designate the remaining required inputs, namely ``first_stage_variables`` and ``second_stage_variables``. Below, we present two separate cases.
+    Additional NLP optimizers to be used in the event the primary
+    subordinate local and global optimizers passed
+    to the PyROS :meth:`~pyomo.contrib.pyros.PyROS.solve` method
+    do not successfully solve a subproblem to an appropriate termination
+    condition can be provided through the optional keyword arguments
+    ``backup_local_solvers`` and ``backup_global_solvers``.
```

Comment:
"condition can be provided through the optional keyword arguments ``backup_local_solvers`` and ``backup_global_solvers``." --> "condition.  These alternative solvers are provided through the optional keyword arguments ``backup_local_solvers`` and ``backup_global_solvers``."